packages/utils/src/eval-worker.ts

```diff
@@ -1,6 +1,8 @@
-import { readFileSync } from 'fs';
-// eslint-disable-next-line import/no-unassigned-import
+/* eslint-disable import/no-unassigned-import */
+
 import 'ses/lockdown';
```

Comment:
nit: I don't think we need to disable the rule for the entire file.

```suggestion
// eslint-disable-next-line import/no-unassigned-import	
import 'ses/lockdown';
```